app.js

```diff
@@ -35,4 +35,86 @@ app.use((err, _, res, __) => {
   });
 });
 
+app.set("view engine", "ejs");
+// app.use(express.static(path.resolve(__dirname, "./public")));
```

Comment:
To zakomentowane zostało, to jak serwujesz pliki? 
O tą treść zadania chodzi?
Ponizej treść zadania: 
Umieść dowolny obraz w folderze public/avatars i sprawdź, czy rozdawanie statyki działa. Po przejściu po takim URL przeglądarka wyświetli obraz.

http://localhost:<port>/avatars/<nazwa pliku z rozszerzeniem>